cmd/cstor-pool-mgmt/controller/cstorpoolinstance-controller/controller.go

```diff
@@ -70,7 +70,7 @@ func NewCStorPoolController(
 	cStorInformerFactory informers.SharedInformerFactory) *CStorPoolController {
```

Comment:
hmm.. directory name have too many 'controller' words..
how about it to be 'cstor-pool-instance'.. sorry, I would have mentioned in previous comments itself